tracing-appender/src/lib.rs

```diff
@@ -151,7 +151,7 @@
 )]
 use crate::non_blocking::{NonBlocking, WorkerGuard};
 
-use std::io::Write;
+use std::{fmt::Debug, io::Write};
```

Comment:
this looks like it's no longer necessary and can be backed out?
```suggestion
use std::io::Write;
```